_posts/elements/2015-04-15-icons.md

```diff
@@ -1,29 +1,33 @@
 ---
 layout:     element
 title:      Icons
-date:       2015-04-15 00:00:00
+date:       2015-09-15 00:00:00
 
 categories: elements
-tags:       
+tags:
 - atomic
 - pattern
 - icons
 
 slug:       icons
+scssurl:    "components/_icons.scss"
 
 doc:        Styleguide:-Icons
 
-description:  A collection of accessible SVG icons available for using within the edX platform.
+description:  A collection of accessible icons available for using within the edX platform.
```

Comment:
NIT: this should mention that these icons are font-based and I think we should remove the "available for using within the edX platform", since the PL is not in the platform currently. "available for use when building UI with the edX Pattern Library"